test/test.js

```diff
@@ -10,6 +10,12 @@ function sortLeftToRight(a, b) {
 	return 0;
 }
 
+function assertTree(tree, nbElements) {
+	assert.strictEqual(tree._getCount(tree.root), nbElements);
+	assert.strictEqual(tree._isSorted(tree.root), true);
```

Comment:
Deleting tests because you have broken them is really.. bad.

Anyhow, I ran those deleted tests and they all pass, so.  I guess you fixed your bugs.

As the code is now, it is not verifiably correct.